.github/workflows/pr.yml

```diff
@@ -0,0 +1,17 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+      - run: npm ci
+      - run: npm run "lint:check"
+#      - run: npm test
```

Comment:
I like the idea of running tests.

I think we should do one of the following here.

- Remove the comment
- Add test to this workflow

Reply:
waiting for the test branch to be merged so this could be uncommented

Reply:
@SamHuo213 we should enable github action require branch to be up to date to master, i didn't even realize my branch is not up to date with main

Reply:
also after this gets merged we can enable branch protection as such that if the `buidl` job fails it won't let pr get merged.

Reply:
I like both ideas, please add tasks to our channel so someone can pick it up.

Reply:
I know Github PRs can display when checks don't pass as a comment. That should be part of this change.